### ISSUE TITLE
Minimum browser requirements

### ISSUE THREAD

**Issue by tbekolay** [nengo/nengo_gui#749]
We should state these somewhere and test each release with them.

**Comment by rsimmons1** [comment]
I agree, however we may have to only use local testing since travis only supports firefox by default. I tried  using chrome on travis a few times and it never worked out, but we can look back into that, and safari is not supported by linux so travis wouldn't be able to test test it.

**Comment by jgosmann** [comment]
But the more browsers we test, the more likely it is that the remaining browser work too. Especially if the you the same backend (does Chrome also use webkit?)
